### PR TITLE
Completed Level 1 and 2 of Readfile

### PR DESCRIPTION
**Pull request by CollinHovey**
Screenshot from 2022-06-29 16-13-08

### CODE REVIEW

**Comment by CollinHovey** [comment]
Level 3 Solution

Screenshot from 2022-06-29 16-20-31

File: node-fs-readfile/read-any.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+
+for (var x = 2; x < process.argv.length; x ++) {
+  fs.readFile(process.argv[x], 'utf8', (err, data) => {
+    console.log(data);
+  });
+}
```
Comment (scbowler) [inline node-fs-readfile/read-any.js:7]: It only needed to read one file, but great job making it read unlimited files 👍

**Review by scbowler — APPROVED** [review]
Great job! 👍

File: node-fs-readfile/read-any.js (hunk 1 above)
Comment (CollinHovey) [reply node-fs-readfile/read-any.js:7]: ya i assumed that was going to be something later so i just did that when making it

**Pull request merged**